client/gutenberg/extensions/slideshow/save.js

```diff
@@ -0,0 +1,39 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { Component } from '@wordpress/element';
+import classnames from 'classnames';
+
+class SlideshowSave extends Component {
+	render() {
+		const { attributes, className } = this.props;
+		const { align, effect, images } = attributes;
+		const alignClassName = align ? `align${ align }` : null;
+		const classes = classnames( className, alignClassName );
+		return (
+			<div className={ classes } data-effect={ effect }>
+				{ images.map( ( image, index ) => {
+					const { alt, caption, height, id, url, width } = image;
+					return (
+						<div className="wp-block-slideshow_image_container" key={ index }>
+							<img
+								src={ url }
+								alt={ alt }
+								data-is-image
+								data-id={ id }
+								data-height={ height }
+								data-width={ width }
+							/>
+							<figcaption data-is-caption>{ caption }</figcaption>
```

Comment:
should we always have a caption? We could probably only add it if there is a caption with
{caption && <figcaption data-is-caption>{ caption }</figcaption>}

Reply:
The caption handling was resolved in https://github.com/Automattic/wp-calypso/pull/29419/commits/3b5518c3b018775ecb8059347dc6432df23eb5fa